tango/local_workspace.py

```diff
@@ -576,12 +570,7 @@ def registered_run(self, name: str) -> Run:
             raise KeyError(name)
         with SqliteDict(self.step_info_file, flag="r") as d:
             steps_for_run = {}
-            for step_symlink in run_dir.iterdir():
-                if not step_symlink.is_symlink():
-                    continue
-                step_name = str(step_symlink.name)
-                unique_id = str(step_symlink.resolve().name)
-                step_info = d[unique_id]
+            for step_name, step_info in d.items():
```

Comment:
This will run through all steps in sqlite. The old code only ran over all steps that are part of that run.

Reply:
So, should we just create empty links for uncacheable steps?